coreos-base/coreos/coreos-0.0.1.ebuild

```diff
@@ -69,11 +69,9 @@ RDEPEND=">=sys-apps/baselayout-3.0.0"
 # If protocol 1 is installed it must be configured to provide the default
 # implementation based on whether protocol 2 is enabled or not.
 RDEPEND="${RDEPEND}
-	etcd_protocols_1? (
-		dev-db/etcd:1
-		!etcd_protocols_2? ( dev-db/etcdctl )
-	)
-	etcd_protocols_2? ( dev-db/etcd:2 )
+	dev-db/etcdctl
```

Comment:
I see! We need to define an approach about how we're going to deal with CLI tools that live inside container images. I'll start a discussion in a different place.